src/PrestaShopBundle/Form/Admin/Sell/Product/Pricing/ApplicableGroupsType.php

```diff
@@ -107,15 +97,8 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         ], $this->groupByIdChoiceProvider->getChoices());
 
         $builder
-            ->add('currency_id', ChoiceType::class, [
-                'label' => false,
-                'placeholder' => false,
-                'choices' => $currencies,
-                'choice_attr' => $this->currencyByIdChoiceProvider->getChoicesAttributes(),
-                'required' => false,
-                'attr' => [
-                    'data-default-currency-symbol' => $this->defaultCurrencySymbol,
-                ],
```

Comment:
This should be added in `CurrencyChoiceType` even if it's not useful for al use cases it doesn't hurt if it's always here. However, removing it is not ok because some JS code must rely on it It's at least needed for two use cases

Reply:
Updated